### ISSUE TITLE
[REQUEST]

### ISSUE THREAD

**Issue by KMozor** [hydralauncher/hydra#360]
Add a backend for your app. Thanks to it you could do synchronization of saves between different devices.
And after that in the settings you specify the address to the server
And Self-Hosted, please.

**Comment by Magrid0** [comment]
Maybe this could be implemented in a similar as Ludusavi on playnite

**Comment by KMozor** [comment]
That would be nice :D

**Comment by Mateleo** [comment]
Yes I vote for a self hosted solution for this, would be awesome !
